### ISSUE TITLE
[Accessibility]: Avatar URL input is too small on mobile view

### ISSUE THREAD

**Issue by kopancek** [sourcegraph/sourcegraph#35705]
### Audit type

Mobile device navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34114

### Problem description

When navigating the user profile section on a mobile device, if the user wants to change their avatar, they need to change the input for avatar URL. However the input and the avatar image are displayed next to each other. This means that the input ends up showing ~2 characters, which is not very user friendly.
<img width="343" alt="Screenshot 2022-05-19 at 14 44 04" src="https://user-images.githubusercontent.com/9974711/169296093-1e8ce3ec-5db2-424e-9a15-12f9d52a9886.png">

### Expected behavior

On a narrow viewport I would expect the avatar to be shown under the avatar URL input. This would leave more horizontal space for the input itself.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

**Comment by sourcegraph-bot-2** [comment]
Heads up @taylorsperry @jasongornall - the "team/frontend-platform" label was applied to this issue.

**Comment by umpox** [comment]
Related to https://github.com/sourcegraph/sourcegraph/issues/35706
